### DISCUSSION TITLE
A320 autoland

### DISCUSSION THREAD

**Discussion by HappySimmer** [flybywiresim/aircraft#5466] [Help] [answered]
Hi guys,

I downloaded the github A320 experimental mod in the hope that I could use the autoland feature, but whatever I try it is not possible to activate AP1+2 simultaneously. Please advise!

**Comment by ExampleWasTaken** [comment, marked as answer]
The experimental branch is currently on hold until further notice. We will notify you through our #server-announcements when we have integrated the custom FPM and LNAV and make the experimental version available again. Until then please use the development version.
